config/locales/i18n/fr.yml

```diff
@@ -3,6 +3,6 @@ fr:
   i18n:
     language: Langue
     locale:
-      en: English
-      es: Español
-      fr: Français
+      en: anglais
+      es: espagnol
+      fr: français
```

Comment:
These are used in the language picker so that we show "English" as an option when we are displaying the page in French. This change should probably be reverted.